layouts/partials/search.html

```diff
@@ -89,6 +88,26 @@
       }
 
       modalSearchInput.addEventListener("input", modalSearch);
+
+      // Event delegation for tracking link clicks
+      modalSearchResults.addEventListener('click', function(event) {
+        if (event.target.tagName === 'A' && event.target.closest('.link')) {
+          if (window.heap === undefined) {
+            return;
+          }
```

Comment:
nit: I guess we could add this condition before even attaching the eventlistener.

```
if !(window.heap === undefined) {
  modalSearchResults.addEventListener(...)
}
```

Reply:
Updated in the commit below. Also updated it from capturing the full target url to just the target path.